.travis.yml

```diff
@@ -19,6 +19,15 @@ matrix:
       env: TOXENV=docs
     - python: "2.7"
       env: TOXENV=py27
+    - if: tag IS present
+      python: "3.6"
```

Comment:
I was curious why not 3.7 here. But 3.7 requires special setup. So, 3.6 is OK here.